unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/ChatNotificationController.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Linq;
+using Cysharp.Threading.Tasks;
+using UnityEngine;
+using DCL.Interface;
+using DCL;
+
+public class ChatNotificationController : IHUD
+{
+    private DataStore dataStore;
+    private IChatController chatController;
+    private MainChatNotificationsComponentView mainChatNotificationView;
```

Comment:
We should implement an `IMainChatNotificationsComponentView` interface to ease the injection of the dependency and the test implementation of this controller.